pom.xml

```diff
@@ -69,7 +69,7 @@
         <pluginRepository>
             <snapshots>
                 <enabled>false</enabled>
-            </snapshots>
+           </snapshots>
```

Comment:
Formatting changed?

Reply:
Mmm it does not find io.jsonwebtoken.io.GsonSerializer in the groovy test.

Reply:
Anyway the typos problem are resolved now.